crates/host/src/wasmbus/mod.rs

```diff
@@ -416,6 +414,7 @@ impl Bus for Handler {
         let (res_r, mut res_w) = socket_pair()?;
 
         let nats = self.nats.clone();
+        let lattice_prefix = self.lattice_prefix.clone();
```

Comment:
```suggestion
        let lattice_prefix = &self.lattice_prefix;
```

Reply:
This still does not look addressed, but I don't mind fixing that in a follow-up

Reply:
Look below, the usage is in an `async move`. Try to compile this as a ref and you'll get a lifetime error

Reply:
You're right, I've missed that part